src/utils/getMonthlyResultsByOption.js

```diff
@@ -83,6 +83,31 @@ export function getMonthlyResultsByOption ({
   return monthlyResultsByOption
 }
 
+export function getYearlyResultsByOption (arr) {
```

Comment:
I'd like you to take a quick look at the quick function I built. There are a few things that I am unsure about and want to run by you.

1. Are you OK with the function using monthly results to calculate yearly as opposed to doing it organically?
2. I'm not sure if this would be the right file for the function. There's obviously something wrong about exporting a `getYearlyResultsByOption` function from a getMonthlyResultsByOption file. Not sure if we should make a new file or change the name of this file to something more generalized.
3. I need to look into this further but for some reason I was getting mortgage payments on the table on the 16th year for the 15 year scenario which doesn't make sense. I'm not quite sure if this is a problem with my function, or the original `getMonthlyResultsByOption()`
4. I noticed the objects returned by `getMonthlyResultsByOption()` had a `budget` property. I ignored it here because we don't show it on the amortization table. What should we do with it?

Reply:
1. I like it. What if we just called `getMonthlyResultsByOption` internally here, rather than passing in a computed array?

2. Yes, you can rename this file, something like `TimeSeriesResultsByOption.getMonthly`

3. I used `budget` to check my work. pmt + amount invested should always == budget.

4. 🤷‍♂